io.catenax.battery.battery_pass/3.0.1/BatteryPass.ttl

```diff
@@ -0,0 +1,920 @@
+    bamm:see <https%3A%2F%2Feur-lex.europa.eu%2Flegal-content%2FEN%2FTXT%2F%3Furi%3DCELEX%3A52020PC0798>;
+    bamm:characteristic :PowerCharacteristic.
+:powerFade a bamm:Property;
+    bamm:name "powerFade";
+    bamm:preferredName "maximum allowed battery power"@en;
+    bamm:description "Maximum allowed battery power (W) of the battery is describing a business requirement."@en;
+    bamm:see <https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=CELEX:52020PC0798>;
+    bamm:characteristic :PowerFadeCharacteristic;
+    bamm:exampleValue "23.0"^^xsd:float.
+:powerCapabilityAt20Charge a bamm:Property;
+    bamm:name "powerCapabilityAt20Charge";
+    bamm:preferredName "power capability at 20 charge"@en;
+    bamm:description "Power (capability) at 20% state of charge. Description from the Regulation is describing a regulatory requirement."@en;
+    bamm:characteristic :PowerCharacteristic.
+:powerCapabilityAt80Charge a bamm:Property;
+    bamm:name "powerCapabilityAt80Charge";
+    bamm:preferredName "power capability at 80 charge"@en;
+    bamm:description "Power (capability) at 80% state of charge. Description from the Regulation is describing a regulatory requirement."@en;
+    bamm:characteristic :PowerCharacteristic.
+:originalPowerCapability a bamm:Property;
+    bamm:name "originalPowerCapability";
+    bamm:preferredName "original power capability"@en;
+    bamm:description "Performance ability of the high voltage battery i.e. the amount of energy that a battery is capable to provide over a given period of time is describing a regulatory requirement."@en;
+    bamm:characteristic :PowerCharacteristic.
+:originalPowerCapabilityLimits a bamm:Property;
+    bamm:name "originalPowerCapabilityLimits";
+    bamm:preferredName "original power capability limits"@en;
+    bamm:description "Performance ability of the high voltage battery according to limits i.e. how much power it can provide within given limits is describing a regulatory requirement."@en;
+    bamm:characteristic :PowerCharacteristic.
+:minVoltage a bamm:Property;
+    bamm:name "minVoltage";
+    bamm:preferredName "min voltage"@en;
+    bamm:description "Value of the minimal voltage the battery is rated for is describing a regulatory requirement."@en;
+    bamm:characteristic :VoltCharacteristic;
+    bamm:exampleValue "2"^^xsd:float.
+:nominalVoltage a bamm:Property;
+    bamm:name "nominalVoltage";
+    bamm:preferredName "nominal voltage"@en;
+    bamm:description "Value of the nominal voltage the battery is rated for is describing a regulatory requirement."@en;
+    bamm:characteristic :VoltCharacteristic;
+    bamm:exampleValue "4.3"^^xsd:float.
+:maxVoltage a bamm:Property;
+    bamm:name "maxVoltage";
+    bamm:preferredName "max voltage"@en;
+    bamm:description "Value of the maximum voltage the battery is rated for is describing a regulatory requirement."@en;
+    bamm:characteristic :VoltCharacteristic;
+    bamm:exampleValue "6"^^xsd:float.
```

Comment:
why using float with the VoltCharacteristic but decimal with the OhmCharacteristic. Wouldn't using a consistent datatype be more appropriate?

Reply:
all floats are converted to decimal